### ISSUE TITLE
Types are not exported in the package.json file

### ISSUE THREAD

**Issue by gege251** [alexplumb/material-ui-linkify#7]
It would be great, if the type definition file would be available for the library users. I think, you'd need to put something like this in your package.json file:
```
  "types": "dist/index.d.ts",
```

Also, your build script doesn't generate type definition files, which could be solved by a separate script for that:
```
    "build:types": "tsc --emitDeclarationOnly",
```

**Comment by 9point6** [comment]
Any chance we could get this done @alexplumb ?

**Comment by alexplumb** [comment]
I'm happy to accept a pull request for this @9point6 - I'll publish a new version as well as bump the peer dep to allow for material ui 5

**Comment by 9point6** [comment]
@alexplumb raised #21!

**Comment by alexplumb** [comment]
Sorry for the wait on this - resolved in `3.0.0`
